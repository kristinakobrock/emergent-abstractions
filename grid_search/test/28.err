  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 19.31it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 22.07it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 21.73it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 23.01it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 23.50it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 17.97it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 19.06it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 22.92it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 23.49it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 25.10it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.24it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 24.71it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 27.39it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.74it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 27.52it/s]100%|############################################################################################| 50/50 [00:02<00:00, 29.51it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.67it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 22.05it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.69it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 24.54it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 22.45it/s]100%|############################################################################################| 13/13 [00:00<00:00, 22.64it/s]
