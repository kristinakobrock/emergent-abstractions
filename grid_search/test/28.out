# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6995814442634583, "acc": 0.5887734293937683, "length": 3.2105140686035156, "mode": "train", "epoch": 1}
{"loss": 0.6712188720703125, "acc": 0.6011719107627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6633904576301575, "acc": 0.6247076988220215, "length": 1.9184213876724243, "mode": "train", "epoch": 2}
{"loss": 0.6632282137870789, "acc": 0.6226562857627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6586163640022278, "acc": 0.6272003650665283, "length": 1.5812138319015503, "mode": "train", "epoch": 3}
{"loss": 0.6610571146011353, "acc": 0.622851550579071, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6476372480392456, "acc": 0.6306365728378296, "length": 1.9009114503860474, "mode": "train", "epoch": 4}
{"loss": 0.6457064151763916, "acc": 0.6351073980331421, "length": 1.9140625, "mode": "test", "epoch": 4}
{"loss": 0.626236617565155, "acc": 0.6560055017471313, "length": 2.0193307399749756, "mode": "train", "epoch": 5}
{"loss": 0.6505404710769653, "acc": 0.628631591796875, "length": 1.6484375, "mode": "test", "epoch": 5}
{"loss": 0.6173626780509949, "acc": 0.6652304530143738, "length": 1.9017645120620728, "mode": "train", "epoch": 6}
{"loss": 0.6510035395622253, "acc": 0.637866199016571, "length": 1.90625, "mode": "test", "epoch": 6}
{"loss": 0.6079075336456299, "acc": 0.6732280850410461, "length": 1.9476447105407715, "mode": "train", "epoch": 7}
{"loss": 0.6496692299842834, "acc": 0.627331554889679, "length": 1.80078125, "mode": "test", "epoch": 7}
{"loss": 0.6009882688522339, "acc": 0.6761853694915771, "length": 2.058741807937622, "mode": "train", "epoch": 8}
{"loss": 0.6604498624801636, "acc": 0.615710437297821, "length": 1.54296875, "mode": "test", "epoch": 8}
{"loss": 0.5940282344818115, "acc": 0.6805626749992371, "length": 2.2056329250335693, "mode": "train", "epoch": 9}
{"loss": 0.6387181878089905, "acc": 0.636639416217804, "length": 2.0859375, "mode": "test", "epoch": 9}
{"loss": 0.5862909555435181, "acc": 0.6872307658195496, "length": 2.415104389190674, "mode": "train", "epoch": 10}
{"loss": 0.6289635300636292, "acc": 0.6487731337547302, "length": 2.4296875, "mode": "test", "epoch": 10}
{"loss": 0.577650249004364, "acc": 0.6976847052574158, "length": 2.7259864807128906, "mode": "train", "epoch": 11}
{"loss": 0.6185283660888672, "acc": 0.660412609577179, "length": 2.5703125, "mode": "test", "epoch": 11}
{"loss": 0.5576606392860413, "acc": 0.7186694145202637, "length": 2.932743787765503, "mode": "train", "epoch": 12}
{"loss": 0.6034400463104248, "acc": 0.6851135492324829, "length": 3.30859375, "mode": "test", "epoch": 12}
{"loss": 0.5317585468292236, "acc": 0.7376748323440552, "length": 3.3797898292541504, "mode": "train", "epoch": 13}
{"loss": 0.596329391002655, "acc": 0.6904540657997131, "length": 3.35546875, "mode": "test", "epoch": 13}
{"loss": 0.5129983425140381, "acc": 0.7464838624000549, "length": 3.414952039718628, "mode": "train", "epoch": 14}
{"loss": 0.5888901948928833, "acc": 0.699951171875, "length": 3.51953125, "mode": "test", "epoch": 14}
{"loss": 0.4978102147579193, "acc": 0.7562437057495117, "length": 3.4865686893463135, "mode": "train", "epoch": 15}
{"loss": 0.5876331329345703, "acc": 0.713085949420929, "length": 3.671875, "mode": "test", "epoch": 15}
{"loss": 0.4794863164424896, "acc": 0.7661982178688049, "length": 3.5664310455322266, "mode": "train", "epoch": 16}
{"loss": 0.5494456887245178, "acc": 0.7364318370819092, "length": 3.71875, "mode": "test", "epoch": 16}
{"loss": 0.4543490707874298, "acc": 0.7854822874069214, "length": 3.8092575073242188, "mode": "train", "epoch": 17}
{"loss": 0.5390607118606567, "acc": 0.737322986125946, "length": 3.99609375, "mode": "test", "epoch": 17}
{"loss": 0.43237054347991943, "acc": 0.7972708940505981, "length": 3.945981025695801, "mode": "train", "epoch": 18}
{"loss": 0.5252677202224731, "acc": 0.7644531726837158, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4079322814941406, "acc": 0.8129949569702148, "length": 3.9720678329467773, "mode": "train", "epoch": 19}
{"loss": 0.4878595173358917, "acc": 0.7794922590255737, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.39018553495407104, "acc": 0.8248608112335205, "length": 3.983520984649658, "mode": "train", "epoch": 20}
{"loss": 0.46914228796958923, "acc": 0.7884765267372131, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3706875443458557, "acc": 0.8386461734771729, "length": 3.989043951034546, "mode": "train", "epoch": 21}
{"loss": 0.45840853452682495, "acc": 0.8003906607627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.366423636674881, "acc": 0.8404150605201721, "length": 3.9939565658569336, "mode": "train", "epoch": 22}
{"loss": 0.46132561564445496, "acc": 0.7982421517372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.34833669662475586, "acc": 0.8527794480323792, "length": 3.9943618774414062, "mode": "train", "epoch": 23}
{"loss": 0.47563672065734863, "acc": 0.7894530892372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3417673707008362, "acc": 0.8550223112106323, "length": 3.994636058807373, "mode": "train", "epoch": 24}
{"loss": 0.4592329263687134, "acc": 0.8013672232627869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3340759873390198, "acc": 0.8595269322395325, "length": 3.9955101013183594, "mode": "train", "epoch": 25}
{"loss": 0.44215667247772217, "acc": 0.8132811784744263, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.32610952854156494, "acc": 0.8661116361618042, "length": 3.9965078830718994, "mode": "train", "epoch": 26}
{"loss": 0.4681747257709503, "acc": 0.7994140982627869, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.31500738859176636, "acc": 0.8723061084747314, "length": 3.996666669845581, "mode": "train", "epoch": 27}
{"loss": 0.44879987835884094, "acc": 0.8156250715255737, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3018198013305664, "acc": 0.8769966959953308, "length": 3.997600555419922, "mode": "train", "epoch": 28}
{"loss": 0.4259529709815979, "acc": 0.8271484971046448, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.29065442085266113, "acc": 0.8831250667572021, "length": 3.998009443283081, "mode": "train", "epoch": 29}
{"loss": 0.42578816413879395, "acc": 0.8253906965255737, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.28718632459640503, "acc": 0.8875862956047058, "length": 3.999040365219116, "mode": "train", "epoch": 30}
{"loss": 0.4308003783226013, "acc": 0.8273438215255737, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.2747498154640198, "acc": 0.895597517490387, "length": 3.9976420402526855, "mode": "train", "epoch": 31}
{"loss": 0.41549208760261536, "acc": 0.8333984613418579, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2760368585586548, "acc": 0.8948130011558533, "length": 3.9987473487854004, "mode": "train", "epoch": 32}
{"loss": 0.4041339159011841, "acc": 0.8464843034744263, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.26509883999824524, "acc": 0.9013273119926453, "length": 3.9998104572296143, "mode": "train", "epoch": 33}
{"loss": 0.41543564200401306, "acc": 0.8392577767372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.2695988714694977, "acc": 0.8987355828285217, "length": 3.9984095096588135, "mode": "train", "epoch": 34}
{"loss": 0.40150508284568787, "acc": 0.8460937142372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.25552862882614136, "acc": 0.9051513075828552, "length": 3.9998393058776855, "mode": "train", "epoch": 35}
{"loss": 0.42642900347709656, "acc": 0.8367186784744263, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.2582867741584778, "acc": 0.9033504724502563, "length": 3.998810291290283, "mode": "train", "epoch": 36}
{"loss": 0.4148901402950287, "acc": 0.8392577767372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.24818521738052368, "acc": 0.9093462228775024, "length": 3.998764991760254, "mode": "train", "epoch": 37}
{"loss": 0.4239565432071686, "acc": 0.8458985090255737, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.2612462341785431, "acc": 0.9041782021522522, "length": 3.9998042583465576, "mode": "train", "epoch": 38}
{"loss": 0.41967907547950745, "acc": 0.83984375, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2514720559120178, "acc": 0.91103595495224, "length": 3.999767780303955, "mode": "train", "epoch": 39}
{"loss": 0.3948195278644562, "acc": 0.8517578840255737, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2341226190328598, "acc": 0.9211032390594482, "length": 3.9997668266296387, "mode": "train", "epoch": 40}
{"loss": 0.4075586497783661, "acc": 0.8478516340255737, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.23549631237983704, "acc": 0.9185444116592407, "length": 3.9996602535247803, "mode": "train", "epoch": 41}
{"loss": 0.40047594904899597, "acc": 0.8580078482627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.2308194786310196, "acc": 0.9211757779121399, "length": 3.999966621398926, "mode": "train", "epoch": 42}
{"loss": 0.42316707968711853, "acc": 0.8531250357627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2668460011482239, "acc": 0.9046395421028137, "length": 3.999185800552368, "mode": "train", "epoch": 43}
{"loss": 0.4069182872772217, "acc": 0.8498047590255737, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2599881589412689, "acc": 0.909048855304718, "length": 3.9987757205963135, "mode": "train", "epoch": 44}
{"loss": 0.3888988196849823, "acc": 0.8576172590255737, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.24706599116325378, "acc": 0.9129661917686462, "length": 3.9995102882385254, "mode": "train", "epoch": 45}
{"loss": 0.41559478640556335, "acc": 0.8472656607627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2349708080291748, "acc": 0.9193443655967712, "length": 3.999718189239502, "mode": "train", "epoch": 46}
{"loss": 0.39622408151626587, "acc": 0.8597656488418579, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.22951212525367737, "acc": 0.921549379825592, "length": 3.9957664012908936, "mode": "train", "epoch": 47}
{"loss": 0.36739543080329895, "acc": 0.875, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.23929402232170105, "acc": 0.9168069958686829, "length": 3.9999775886535645, "mode": "train", "epoch": 48}
{"loss": 0.4212612211704254, "acc": 0.8589844107627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.23855151236057281, "acc": 0.9157150387763977, "length": 3.9999778270721436, "mode": "train", "epoch": 49}
{"loss": 0.39004138112068176, "acc": 0.8625000715255737, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.23047098517417908, "acc": 0.9198247790336609, "length": 3.9996142387390137, "mode": "train", "epoch": 50}
{"loss": 0.3969123363494873, "acc": 0.8542968034744263, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.22966472804546356, "acc": 0.9225112795829773, "length": 3.999986171722412, "mode": "train", "epoch": 51}
{"loss": 0.40013235807418823, "acc": 0.8625001311302185, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.22704973816871643, "acc": 0.9188708662986755, "length": 3.999979257583618, "mode": "train", "epoch": 52}
{"loss": 0.3928656578063965, "acc": 0.859375, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2273244857788086, "acc": 0.9211530089378357, "length": 3.999595880508423, "mode": "train", "epoch": 53}
{"loss": 0.3637872338294983, "acc": 0.8763671517372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.216182142496109, "acc": 0.9256067276000977, "length": 3.999999523162842, "mode": "train", "epoch": 54}
{"loss": 0.37794822454452515, "acc": 0.8757811784744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.21209996938705444, "acc": 0.9273054003715515, "length": 3.9999923706054688, "mode": "train", "epoch": 55}
{"loss": 0.36876747012138367, "acc": 0.8767577409744263, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.21154667437076569, "acc": 0.92948979139328, "length": 3.9999911785125732, "mode": "train", "epoch": 56}
{"loss": 0.3615092635154724, "acc": 0.8845702409744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.20165905356407166, "acc": 0.9339806437492371, "length": 3.999999523162842, "mode": "train", "epoch": 57}
{"loss": 0.37900903820991516, "acc": 0.8751953840255737, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.20761209726333618, "acc": 0.9306427836418152, "length": 3.999992847442627, "mode": "train", "epoch": 58}
{"loss": 0.3801129460334778, "acc": 0.876953125, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.21041832864284515, "acc": 0.9280338287353516, "length": 3.999997854232788, "mode": "train", "epoch": 59}
{"loss": 0.3730051517486572, "acc": 0.8757811188697815, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.20530077815055847, "acc": 0.9323370456695557, "length": 3.9999167919158936, "mode": "train", "epoch": 60}
{"loss": 0.35586652159690857, "acc": 0.8869141340255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7589705586433411
